### PR TITLE
Add minutes for 2024-12-11

### PR DESCRIPTION
**Pull request by sjn**
Initial version for review

### CODE REVIEW

File: meetings/cpansec-minutes-2024-11-15.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+### Ongoing vulnerabilities
+*   @stigtsp - Critical PAUSE vuln fixed quickly by andk https://github.com/andk/pause/security/advisories/GHSA-3j5w-wmwq-p6cw
+*   @stigtsp - No movement around Mojo's secure token/secrets issues; Volunteers needed
+*   @stigtsp - One more vuln known, needs a volunteer; @timlegge may take a look
+
+### Secure by Default
+*   @stigtsp - HTTP::Tiny 0.090 merged into Perl CORE, using system cacerts instead of Mozilla::CA
+
+### CPAN Author's Security Policy Template/Guidelines
+*   @rrwo - working on initial draft
```
Comment (robrwo) [inline meetings/cpansec-minutes-2024-11-15.md]: I'm @robrwo in later minutes.

**Review by robrwo — COMMENTED** [review]
A nitpick and a question but otherwise alright.

File: meetings/cpansec-minutes-2024-12-11.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    * Find ways to make CPAN tooling to support Package URLs, so others can correctly state their requirements on CPAN packages, and that external ecosystems (e.g. Deb or RPM package indices) and databases (e.g. MITRE CVE) can refer to CPAN packages in an ecosystem-independent way.
+    * Introduce SBOMs into the CPAN toolchain and services, so businesses and other users can comply with regulation without too much cost
+    * Find a way to make CPANSec's work sustainable. We expect a large increase in activity as a consequence of the Cyber Resilience Act coming into effect Dec 10th 2024. This may include the possiblity as setting up CPANSec as a Steward on behalf of the CPAN ecosystem.
+    * In the first few months, we'll have to look into ways to help the Perl/CPAN toolchain community get up to speed on the the coming consequences of the CRA, in such a way that necessary decisions can be made. A natural deadline for this is the PTS in Leipzig in May.
+
+### Other
+*   @sjn – let's set up git commit messages on IRC again!
```
[the remark this answers is not in the document]
Comment (sjn) [reply meetings/cpansec-minutes-2024-12-11.md:86]: No it wasn't; I'll remove it!

**Review by timlegge — APPROVED** [review]
lgtm

**Review by robrwo — APPROVED** [review]

**Pull request merged**